en/peripherals/parachute.md

```diff
@@ -27,50 +28,85 @@ Below are a few considerations when using parachutes:
 ## Parachute Setup
 
 Flight termination (and hence parachute deployment) may be triggered by safety checks such as RC Loss, geofence violation, and so on, from attitude triggers and other failure detector checks, or by a command from a ground station.
-During flight termination PX4 sets PWM outputs to their "failsafe" values (failsafe values turn off motors, but may be used to turn on/trigger the parachute.
+During flight termination PX4 sets PWM outputs to their "failsafe" values (failsafe values turn off motors, but may be used to turn on/trigger the parachute).
+If a MAVLink parachute is connected and healthy, a command will be sent to activate it.
 
 Parachute setup therefore involves:
-- Connecting the parachute to unused outputs and setting their failsafe values to levels that will trigger the parachute (using appropriate parameters).
 - Configuring [flight termination](../advanced_config/flight_termination.md) as the appropriate action for those safety and failure cases where the parachute should be deployed.
+- Configure PX4 output levels to disable motors on failsafe.
+- Configure PX4 to deploy the parachute (set PWM output levels or send MAVLink parachute deploy command).
 
-The setup below explains how you might configure a quad multicopter with motors on MAIN 1-4 and a parachute on MAIN PWM port 7.
-The PWM values mentioned are for the *Fruity Chutes* parachute we tested.
+### Enable Flight Termination
 
-:::tip
-For the first test, try on the bench, without the props and with an unloaded parachute device!
+To enable flight termination:
+- Set [Safety](../config/safety.md) action to *Flight termination* for checks where you want the parachute to trigger.
+- Set [Failure Detector](../config/safety.md#failure-detector) pitch angles, roll angles and time triggers for crash/flip detection, and disable the failure/IMU timeout circuit breaker (i.e. set [CBRK_FLIGHTTERM=0](../advanced_config/parameter_reference.md#CBRK_FLIGHTTERM)).
+  
+:::note
+You can also configure an [external Automatic Trigger System (ATS)](../config/safety.md#external-automatic-trigger-system-ats) for failure detection.
 :::
+  
+### Motor Failsafe Settings
+
```

Comment:
@bkueng The original docs here were very much based around a particular example using the PWM MAIN output 7 on a quad with motors on MAIN1-4. You essentially had to manually set the FAIL values for each of the outputs.

I'm "part" of the way to restructuring this to work for actuators too.
1. Do we need to think about motor "FAIL" settings? My guess is that these are automatically set to disarmed values as part of normal actuator setup (?).
2. The parachute can go on any unused output, and with actuators I can set where it should be:
   ![image](https://user-images.githubusercontent.com/5368500/178655804-16c55d2c-556e-4499-abf5-3f3250c66939.png)
   
   But how do I set the failsafe value?
3. What happens if the system is using UAVCAN? ie. for ESCs and to trigger the parachute. What do I do in this case?

Reply:
@bkueng ping ^^^

Reply:
> Do we need to think about motor "FAIL" settings? My guess is that these are automatically set to disarmed values as part of normal actuator setup (?).

Yes it's automatic, you typically don't need to care.

> But how do I set the failsafe value? Or does PX4 know that for a parachute maximum means "trigger" and minimum/disarmed means "wait"? So values get set appropriately for the function?

It goes to the maximum value. Yes it's per function. See the pararm description for https://docs.px4.io/main/en/advanced_config/parameter_reference.html#PWM_AUX_FUNC1.

> What happens if the system is using UAVCAN? ie. for ESCs and to trigger the parachute. What do I do in this case?

It works the same way. What do you mean by ESC?

Reply:
Thanks. Ignore the "for escs" part. That was when I was thinking that motors in failsafe have another state than disarmed . Ignore me.